File: src/main/java/me/delivery/domain/store/model/entity/Store.java
package me.delivery.domain.store.model.entity;

import javax.persistence.Entity;

import lombok.Getter;
import lombok.Setter;
import me.delivery.domain.user.model.entity.BaseEntity;

@Entity
@Getter
@Setter
public class Store extends BaseEntity {
    private String name;
    private StoreStatus status;
}

File: src/main/kotlin/me/delivery/domain/store/model/entity/Store.kt
package me.delivery.domain.store.model.entity;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
@Getter
@Setter
public class Store {
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;
    private String name;
    private StoreStatus status;

    public void a() {
        System.out.println("");
    }

}
